includes/class-wc-gateway-payfast.php

```diff
@@ -419,7 +419,7 @@ public function generate_payfast_form( $order_id ) {
 
 			// Custom strings.
 			'custom_str1'      => self::get_order_prop( $order, 'order_key' ),
-			'custom_str2'      => 'WooCommerce/' . WC_VERSION . '; ' . get_site_url(),
+			'custom_str2'      => 'WooCommerce/' . WC_VERSION . '; ' . urlencode( get_site_url() ),
```

Comment:
Getting a PHPCS warning here suggesting we use `rawurlencode` instead
```suggestion
			'custom_str2'      => 'WooCommerce/' . WC_VERSION . '; ' . rawurlencode( get_site_url() ),
```